### ISSUE TITLE
Sharing dashboard items between accounts and projects

### ISSUE THREAD

**Issue by alexkim205** [PostHog/posthog#6292]
## Is your feature request related to a problem?

Relevant thread: https://posthogusers.slack.com/archives/CT7HXDEG3/p1632435699279000

There was recently a feature request for the ability to export and import dashboard items/dashboards that could be shared as a "template" across different projects and accounts. 

Google Analytics has a similar concept where they've created an entire community around sharing/ranking dashboards - https://analytics.google.com/analytics/gallery/#posts/search/

One of our users explaining how they would use this feature.

>  The idea here is this; many companies measure similar KPI's even if the data powering those KPI's are different (think conversion rate = [conversion] / [impression]. I could use "orders" or "video plays" or "clicks" for that Dashboard that is showing "conversions" by UTM_SOURCE.   If someone has already built this report, I can literally download/export/browser -> upload (to my post hog site) and have it all ready to go (and if the values's are miss-matching or not available, i.e. my website doesn't have "video plays", then I can swap out the value. 
> ...
> Ideally, once you've added a dashboard, you could "share" the dashboard to the community, or not.

## Describe the solution you'd like

MVP: Ability to export and import dashboard items.
Potential next steps:
- Expand to exporting and importing entire dashboards
- "Templates" for insights or collection of insights that can be shared
- Enable community for sharing and exploring dashboards.
- Starter templates to make onboarding/discovery easier

## Describe alternatives you've considered

Recreating insights manually across different projects, which is time consuming and tedious.

**Comment by lsmith77** [comment]
I think an MVP step would be to just export to JSON. This could also open the door for users to build "insights" generators.

**Comment by Hronom** [comment]
Yes having export to json and import from json will helps to move dashboards between projects and accounts. Same as in grafana https://grafana.com/docs/grafana/latest/dashboards/share-dashboards-panels/#export-a-dashboard-as-json
